bin/postgres_common/postgres/reload.sh

```diff
@@ -13,9 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-source /opt/cpm/bin/common_lib.sh
+CRUNCHY_DIR=${CRUNCHY_DIR:-'/opt/crunchy'}
+source $CRUNCHY_DIR/bin/common_lib.sh
 enable_debugging
 
-source /opt/cpm/bin/setenv.sh
-# source /opt/cpm/bin/custom-configs.sh
+source $CRUNCHY_DIR/bin/postgres/setenv.sh
+# source $CRUNCHY_DIR/bin/postgres/custom-configs.sh
```

Comment:
If a line of code is commented out, and there's no reason given as to why it's commented out, please remove it.